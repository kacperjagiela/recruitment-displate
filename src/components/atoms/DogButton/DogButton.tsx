import { Button } from '@chakra-ui/button';

interface Props {
    buttonText: string;
    onClick: () => void;
    variant?: string;
}

const DogButton: React.FC<Props> = ({ buttonText, onClick, variant }: Props) => (
    <Button colorScheme="blue" onClick={onClick} textTransform="capitalize" textStyle="h4" variant={variant || 'solid'}>
        {buttonText}
    </Button>
);

export default DogButton;
