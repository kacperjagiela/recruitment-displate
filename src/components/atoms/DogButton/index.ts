export { default as Button } from './DogButton';
