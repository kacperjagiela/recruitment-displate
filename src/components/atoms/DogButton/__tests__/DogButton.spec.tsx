import React from 'react';
import { render } from '@testing-library/react';

import DogButton from '../DogButton';

describe('<DogButton />', () => {
    it('renders correctly', () => {
        const { container } = render(<DogButton buttonText="" onClick={jest.fn} />);

        expect(container).toMatchSnapshot();
    });
});
