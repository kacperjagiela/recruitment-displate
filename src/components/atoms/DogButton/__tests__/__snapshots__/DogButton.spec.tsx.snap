// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DogButton /> renders correctly 1`] = `
<div>
  <button
    class="chakra-button css-omiz5g"
    type="button"
  >
    
  </button>
</div>
`;
